docs/pages/desktop-access/reference/sessions.mdx

```diff
@@ -87,4 +76,14 @@ player supports toggling between play and pause, but does not support seeking to
 a specific point in the stream, rewinding, or restarting playback when the end
 of the stream is reached. To replay a session, refresh the page.
 
+## Log rotation
```

Comment:
```suggestion
## Storage
```

I recommend changing the title here because "log rotation" sounds like something you have to do manually, and that's not really the case here.